bluesky_database/backend/REDIS_SETUP.md

```
@@ -0,0 +1,120 @@
+chmod 600 secrets/redis_password.txt
+```
+
+Notes:
+- Do not commit `secrets/redis_password.txt` to source control.
+- Consider adding a `.gitignore` in `backend/secrets/` with `redis_password.txt` to prevent accidental commits.
+
+## 2) Configuration details
+
+### redis.conf
+- Network binding is set to `127.0.0.1` to illustrate secure defaults:
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)